app/etc/pulp/server.yaml

```diff
@@ -42,3 +42,16 @@
 #     pulp.app:
 #       level: INFO
 
+# Broker configuration
+#
+# `broker`: Broker configuration for Pulp. By default, Pulp uses the Qpid
+# broker without SSL. It is possible to change this by providing a different
+# broker string. The remaining options are used to configure SSL settings.
+#
+# broker:
+#   url: qpid://localhost/
+#   celery_require_ssl: false
+#   cacert: /etc/pki/pulp/qpid/ca.crt
+#   keyfile: /etc/pki/pulp/qpid/client.crt
+#   certfile: /etc/pki/pulp/qpid/client.crt
```

Comment:
Nitpicking but sure would be nice if the 3 ssl properties were named more consistency.

Consider:
```
ssl_ca_certificate:
ssl_client_key:
ssl_client_certificate:
```

Might be trying to fix too much and be outside the scope of this PR.